tasks/cnm-to-cma/pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[project]
+name = "cnm2cma"
+version = "0.1.0"
+description = ""
+authors = [
+    { name = "Your Name", email = "you@example.com" }
+]
+readme = "README.md"
+requires-python = ">=3.12,<4.0"
+dependencies = [
+    "boto3-stubs (==1.41.5)",
+    "cumulus-process==1.6.0"
+]
+
+[tool.poetry]
```

Comment:
Recommend removing line 15 and 16 since we are not building wheels, sdist or binaries for this module but will do a zip file for lambda.